core/src/main/java/hudson/model/Run.java

```diff
@@ -538,7 +538,8 @@ public boolean isBuilding() {
      * @return true if after started and before completed.
      * @since 1.538
      */
-    protected boolean isInProgress() {
+    @Exported
+    public boolean isInProgress() {
```

Comment:
Is the change of modifier actually required? This will be incompatible (at least source, perhaps binary) for Pipeline: https://github.com/jenkinsci/workflow-job-plugin/blob/8def1a5f3944cb5d68ceb955b541a83897b13f98/src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java#L780-L781

Reply:
Yes it is required :/